hasjob/views/index.py

```diff
@@ -383,6 +383,15 @@ def my_posts():
 
 
 @csrf.exempt
+@app.route('/applied', subdomain='<subdomain>')
+@app.route('/applied')
+@lastuser.requires_login
+def applied():
+    basequery = JobPost.query.join(JobApplication).filter(JobApplication.user == g.user)
```

Comment:
This will load the full contents of JobApplication, IIRC. You have to defer loading of all columns except `id` and `user_id` using `.options(*defer(<column_object>))`. There's also a mechanism to defer all in recent versions of SQLAlchemy.